ocs_ci/utility/utils.py

```diff
@@ -954,6 +954,34 @@ def ocsci_log_path():
     )
 
 
+def get_testrun_name():
+    """
+    Prepare testrun ID for Polarion (and other reports).
+
+    Return config.REPORTING["polarion"]["testrun_id"], if configured.
+    Otherwise prepare testrun ID based on Upstream/Downstream information,
+    OCS version and used markers.
+
+    Returns:
+        str: String containing testrun ID
+    """
+    markers = config.RUN['cli_params'].get('-m', '').replace(" ", "-")
+    us_ds = config.REPORTING.get("us_ds")
+    if config.REPORTING["polarion"].get("testrun_id"):
+        testrun_id = config.REPORTING["polarion"]["testrun_id"]
+    elif markers:
+        testrun_id = f"OCS_{us_ds}_{config.REPORTING.get('testrun_id_part', '')}_{markers}"
+    else:
+        testrun_id = f"OCS_{us_ds}_{config.REPORTING.get('testrun_id_part', '')}"
+    # replace invalid character(s) by '-'
+    testrun_id = testrun_id.translate(
+        str.maketrans(
+            {key: '-' for key in ''' \\/.:*"<>|~!@#$?%^&'*(){}+`,=\t'''}
+        )
+    )
+    return testrun_id
```

Comment:
is testrun_id is unique for every run

Reply:
Now, `testrun_id` (now renamed as `testrun_name`) should be the same for the same configuration.